pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaConsumerManager.java

```diff
@@ -150,4 +150,30 @@ public void run() {
       }.start();
     }
   }
+
+  public static void closeAllConsumers() {
+    try {
+      // Shutdown all consumers
+      synchronized (KafkaConsumerManager.class) {
+        LOGGER.info("Trying to shutdown all the kafka consumers");
+        Iterator<ConsumerAndIterator> consumerIterator =
+            CONSUMER_AND_ITERATOR_FOR_CONFIG.values().iterator();
+
+        while (consumerIterator.hasNext()) {
+          ConsumerAndIterator consumerAndIterator = consumerIterator.next();
+          LOGGER.info("Trying to shutdown consumer/iterator {}", consumerAndIterator.getId());
```

Comment:
what if we simply call the other method releaseConsumerAndIterator for each consumerAndIterator